src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -0,0 +1,290 @@
+difficulty: beginner
+readtime: 5
+back: /docs/quick-starts
+---
+Learn how to set up your first Go project powered by Appwrite.
+{% section #step-1 step=1 title="Create project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+Then, under **Integrate with your server**, add an **API Key** with the following scopes.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/integrate-server.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to read [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% /section %}
+{% section #step-2 step=2 title="Create Go project" %}
+
+Create a go application.
+
+```sh
+mkdir my-app
+cd my-app
+go mod init go-appwrite/main
+```
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Go Appwrite SDK.
+
+```sh
+go get github.com/appwrite/sdk-for-go
+```
+
+{% /section %}
+
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Create a new file called `app.go`, initialize a function, and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```go
+package main
+
+import (
+	"github.com/appwrite/sdk-for-go/appwrite"
+)
+
+func main() {
+	client := appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_ID>"),
+		appwrite.WithKey("<API_KEY>"),
+	)
+}
```

Comment:
In our other quick starts, we use global variables so that we can refer to client or databases in other functions. Let's do the same here. So, this snippet can be:

```go
package main

import (
	"github.com/appwrite/sdk-for-go/appwrite"
	"github.com/appwrite/sdk-for-go/client"
)

var appwriteClient client.Client

func main() {
	appwriteClient = appwrite.NewClient(
		appwrite.WithProject("<PROJECT_ID>"),
		appwrite.WithKey("<API_KEY>"),
	)
}
```

Reply:
I tried doing this, but the problem is Go returns an error when I try this approach.
Error message: `./main.go:17:2: appwriteClient declared and not used`
```
func main() {

	appwriteClient := appwrite.NewClient(
		appwrite.WithProject("<PROJECT_ID>"),
		appwrite.WithKey("<API_KEY>"),
	)

	prepareDatabase()
}

func prepareDatabase() {
	appwriteDatabases := appwrite.NewDatabases(appwriteClient)
}
```